src/main/java/be/aga/dominionSimulator/gui/DomBotSelector.java

```diff
@@ -1,13 +1,10 @@
 package be.aga.dominionSimulator.gui;
```

Comment:
The changes to this file are not as big as they look... I cleaned up some of the inconsistent whitespacing with an automatic reformat.